FestivalPlanner/loader/tests.py

```diff
@@ -7,10 +7,10 @@
 from http import HTTPStatus
 from unittest import skip
 
-# from FilmFestivalPlanner.FilmFestivalLoader.Shared import planner_interface as planner
 from django.test import RequestFactory
 from django.urls import reverse
 
+from festival_planner import debug_tools
 from festival_planner.debug_tools import pr_debug
```

Comment:
Please make one consequent import from the two lines.

Reply:
Done, but it looks a bit weird in any case:

`pr_debug` is (to be) disabled, but called in one of the tests...

That test is skipped anyway, and a bit scary to fix in this context.